securedrop_client/gui/main.py

```diff
@@ -54,9 +55,9 @@ def __init__(self, sdc_home: str):
         """
         super().__init__()
         self.sdc_home = sdc_home
-        self.controller = None
+        self.controller = None  # type: Optional[Client]
 
-        self.setWindowTitle(_("SecureDrop Client {}").format(__version__))
+        self.setWindowTitle(_("SecureDrop Client {}").format(__version__))  # type: ignore
```

Comment:
btw I have this as type: ignore because of the `_()` function which mypy will flag as a name not defined error

Reply:
We might want to add a stub file with `_` included and if this isn't possible, we might want to do `from gettext import gettext as _` just to be explicit. I'd rather not turn off any of our checks if possible.